client/src/actions/index.ts

```diff
@@ -188,87 +189,89 @@ const dispatchDiffExpErrors = (dispatch: any, response: any) => {
   }
 };
 
-const requestDifferentialExpression = (
-  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any -- - FIXME: disabled temporarily on migrate to TS.
-  set1: any,
+const requestDifferentialExpression =
+  (
+    // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any -- - FIXME: disabled temporarily on migrate to TS.
+    set1: any,
+    // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any -- - FIXME: disabled temporarily on migrate to TS.
+    set2: any,
+    num_genes = 50
+    // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any -- - FIXME: disabled temporarily on migrate to TS.
+  ) =>
   // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any -- - FIXME: disabled temporarily on migrate to TS.
-  set2: any,
-  num_genes = 50
-  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any -- - FIXME: disabled temporarily on migrate to TS.
-) => async (dispatch: any, getState: any) => {
-  dispatch({ type: "request differential expression started" });
-  try {
-    /*
+  async (dispatch: any, getState: any) => {
```

Comment:
shouldn't the parameters be typed, eg, `(dispatch: Dispatch, getState: () => RootState)`?

Reply:
This function isn't edited in this PR

Reply:
Sorry, I assumed you were cleaning up the entire file.  As long as the comment remains, all is fine.